taker-electron/src/main/menu.ts

```diff
@@ -0,0 +1,257 @@
+import { app, BrowserWindow, Menu, MenuItemConstructorOptions, shell } from "electron";
+
+interface DarwinMenuItemConstructorOptions extends MenuItemConstructorOptions {
+    selector?: string;
+    submenu?: DarwinMenuItemConstructorOptions[] | Menu;
+}
+
+export default class MenuBuilder {
+    mainWindow: BrowserWindow;
+
+    constructor(mainWindow: BrowserWindow) {
+        this.mainWindow = mainWindow;
+    }
+
+    buildMenu(): Menu {
+        if (
+            process.env.NODE_ENV === "development"
+            || process.env.DEBUG_PROD === "true"
+        ) {
+            this.setupDevelopmentEnvironment();
+        }
+
+        const template = process.platform === "darwin"
+            ? this.buildDarwinTemplate()
+            : this.buildDefaultTemplate();
+
+        const menu = Menu.buildFromTemplate(template);
+        Menu.setApplicationMenu(menu);
+
+        return menu;
+    }
+
+    setupDevelopmentEnvironment(): void {
+        this.mainWindow.webContents.on("context-menu", (_, props) => {
+            const { x, y } = props;
+
+            Menu.buildFromTemplate([
+                {
+                    label: "Inspect element",
+                    click: () => {
+                        this.mainWindow.webContents.inspectElement(x, y);
+                    },
+                },
+            ]).popup({ window: this.mainWindow });
+        });
+    }
+
+    buildDarwinTemplate(): MenuItemConstructorOptions[] {
+        const subMenuAbout: DarwinMenuItemConstructorOptions = {
+            label: "Electron",
+            submenu: [
+                {
+                    label: "About ElectronReact",
```

Comment:
disclaimer: I haven't tested this, so it might be a wrong suggestion 🙃 especially if there's some magic code that replaces that string later on with the name of the app.

isn't this how the menu will be showed and shouldn't we tailor the following? 

```suggestion
            label: "ItchySats",
            submenu: [
                {
                    label: "About ItchySats",
```

Reply:
Good find